Intersect.Editor/Localization/Strings.cs

```diff
@@ -1141,6 +1141,8 @@ public struct CraftsEditor
 
             public static LocalizedString undotitle = @"Undo Changes";
 
+            public static LocalizedString commonevent = @"Common Event:";
```

Comment:
Casing and JSON properties are required for new additions, see other recently added ones (ie Not null and null value handling)

Something we often forget but probably shouldn't.

Reply:
I don't think we use NotNull anymore. Think Panda removed all those when we removes the Jetbrains libraries?

Reply:
Is it gone from the others? 
Might be misremembering that.

Reply:
Not entirely sure I follow here on what you want me to change/add. The casing is the same as the other things around it.